core/src/main/scala/com/evolutiongaming/kafka/flow/kafka/Consumer.scala

```diff
@@ -19,8 +20,11 @@ import scodec.bits.ByteVector
   */
 trait Consumer[F[_]] {
 
+  @deprecated("Use Consumer.subscribe with topics set parameter instead", "0.5.0")
```

Comment:
If we are not using it in KafkaFlow code anymore, may be let's delete it?

Reply:
Yes, I'd like to drop the method too. But it will bread backward compatibility.

Reply:
Done